pkg/cli/images.go

```diff
@@ -127,52 +138,57 @@ func (a *Image) Run(cmd *cobra.Command, args []string) error {
 }
 
 type imagePrint struct {
+	Project    string `json:"project,omitempty"`
 	Name       string `json:"name,omitempty"`
 	Digest     string `json:"digest,omitempty"`
 	Repository string `json:"repository,omitempty"`
 	Tag        string `json:"tag,omitempty"`
 }
 
 type imageContainer struct {
-	Container string
-	Repo      string
-	Tags      []string
-	Digest    string
-	ImageID   string
+	Container  string
+	Repository string
+	Tags       []string
+	Digest     string
+	ImageID    string
 }
 
 type imageContainerPrint struct {
-	Container string
-	Repo      string
-	Tag       string
-	Digest    string
-	ImageID   string
+	Project    string
+	Container  string
+	Repository string
+	Tag        string
+	Digest     string
+	ImageID    string
 }
 
-func printContainerImages(images []apiv1.Image, ctx context.Context, c client.Client, a *Image, args []string, tagToMatch string) error {
+func printContainerImages(images []apiv1.Image, ctx context.Context, c client.Client, a *Image, args []string, tagToMatch string, allProjects bool) error {
 	out := table.NewWriter(tables.ImageContainer, a.Quiet, a.Output)
 
 	if a.Quiet {
 		out = table.NewWriter([][]string{
-			{"Name", "{{if ne .Repo \"\"}}{{.Repo}}:{{end}}{{.Tag}}@{{.Digest}}"},
+			{"Name", "{{if ne .Repository \"\"}}{{.Repository}}:{{end}}{{.Tag}}@{{.Digest}}"},
 		}, a.Quiet, a.Output)
 	}
 
 	for _, image := range images {
+		imageProject := image.Labels[labels.AcornProject]
 		containerImages, err := getImageContainers(c, ctx, image)
 		if err != nil {
 			return err
 		}
 		for _, imageContainer := range containerImages {
 			imageContainerPrint := imageContainerPrint{
-				Container: imageContainer.Container,
-				Repo:      imageContainer.Repo,
-				Digest:    imageContainer.Digest,
-				ImageID:   imageContainer.ImageID,
+				Container:  imageContainer.Container,
+				Repository: imageContainer.Repository,
+				Digest:     imageContainer.Digest,
+				ImageID:    imageContainer.ImageID,
+			}
+			if allProjects { // Same project for all container images
```

Comment:
nit: is this comment correct?

Reply:
I believe so. If we grab an image, and call `imgDetails := ImageDetails(image)`, following `imgDetails.AppImage.ImageData.(Containers|Jobs) `should be a part of the same project as the original image.